upgrade_groupd.sh

```diff
@@ -0,0 +1,409 @@
+kind: OperatorGroup
+metadata:
+  name: template-service-broker-og
+  namespace: template-service-broker
+spec:
+  targetNamespaces:
+    - template-service-broker
+EOF
+
+  oc create -f tsbog.yaml -n template-service-broker
+
+  cat << EOF > tsbcsc.yaml
+apiVersion: operators.coreos.com/v1
+kind: CatalogSourceConfig
+metadata:
+  name: installed-community-template-service-broker
+  namespace: openshift-marketplace
+spec:
+  csDisplayName: Community Operators
+  csPublisher: Community
+  packages: templateservicebroker
+  targetNamespace: template-service-broker
+EOF
+  oc create -f tsbcsc.yaml -n openshift-marketplace
+
+  cat << EOF > tsbsubscription.yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  labels:
+    csc-owner-name: installed-community-template-service-broker
+    csc-owner-namespace: openshift-marketplace
+  name: templateservicebroker
+  namespace: template-service-broker
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: templateservicebroker
+  source: installed-community-template-service-broker
+  sourceNamespace: template-service-broker
+  startingCSV: templateservicebrokeroperator.v0.2.0
+EOF
+  oc create -f tsbsubscription.yaml -n template-service-broker
+
+  echo "waiting for pods running....."
+  sleep 30
+
+cat << EOF > tsb-cr.yaml
+apiVersion: osb.openshift.io/v1alpha1
+kind: TemplateServiceBroker
+metadata:
+  name: template-service-broker
+  namespace: template-service-broker
+EOF
+  oc create -f tsb-cr.yaml -n template-service-broker
+
+  echo "waiting for pods running....."
+  sleep 30
+  if [ !$(oc get po -n template-service-broker | tail -n +2 |grep  -cvE "(Running|Completed)") ] ; then
+    echo -e  "\033[31m TSB install successfully, need a few minutes to wait tsb ready !  \033[0m"
+  else 
+    echo -e "\033[31m TSB installation failed , please manually install TSB on web console!  \033[0m"
+  fi
+}
+
+#DevExp team
+function devExp {
+  echo "Collectting pre_upgrade data for DevExp team.... "
+  echo -e "================== DEVEXP-TEAM  DATA `date` ============================================" >> $RESULT_FILE
+  echo -e "#oc get pods -o yaml -n openshift-image-registry | grep imageID |uniq\n$(oc get pods -o yaml -n openshift-image-registry | grep imageID |uniq)" >> $RESULT_FILE
+  echo -e "#oc get pods -o yaml -n openshift-cluster-samples-operator | grep imageID |uniq\n$(oc get pods -o yaml -n openshift-cluster-samples-operator | grep imageID |uniq)" >> $RESULT_FILE
+  echo -e "#oc get pods -n openshift-image-registry\n$(oc get pods -n openshift-image-registry)" >> $RESULT_FILE
+  echo -e "#oc describe is jenkins -n openshift\n$(oc describe is jenkins -n openshift)" >> $RESULT_FILE
+  echo -e "#oc describe is ruby -n openshift\n$(oc describe is ruby -n openshift)" >> $RESULT_FILE
+
+  echo "================== DEVEXP team's namespace ${DEVEXP_PRO} is created!"
+  oc new-project ${DEVEXP_PRO}
+  echo "=================Preparing upgrade data for DEVEXP team....."
```

Comment:
Could you help add  `oc patch config.samples.operator.openshift.io cluster -p '{"spec": {"skippedImagestreams": ["perl", "mysql"]}}'  --type merge` here